src/resource.js

```diff
@@ -71,12 +72,19 @@ export default class Resource {
    * @returns {String|Array|Object}
    */
   get source() {
```

Comment:
@Fak3 @roll 

both comments have been addressed. Path checking searches for `^/`, and occurences of `..` or `.\.` etc (backslash is being ignored when travesting dirs, so it's first taken out of the path).